datasets/wmt_mlqe_task3/wmt_mlqe_task3.py

```diff
@@ -0,0 +1,252 @@
+                    {
+                        "segment_id": datasets.Sequence(datasets.Value("int32")),
+                        "annotation_start": datasets.Sequence(datasets.Value("int32")),
+                        "annotation_length": datasets.Sequence(datasets.Value("int32")),
+                        "severity": datasets.ClassLabel(names=["minor", "major", "critical"]),
+                        "severity_weight": datasets.Value("float32"),
+                        "category": datasets.ClassLabel(names=_ANNOTATION_CATEGORIES),
+                    }
+                ),
+                "token_annotations": datasets.Sequence(
+                    {
+                        "segment_id": datasets.Sequence(datasets.Value("int32")),
+                        "first_token": datasets.Sequence(datasets.Value("int32")),
+                        "last_token": datasets.Sequence(datasets.Value("int32")),
+                        "token_after_gap": datasets.Sequence(datasets.Value("int32")),
+                        "severity": datasets.ClassLabel(names=["minor", "major", "critical"]),
+                        "category": datasets.ClassLabel(names=_ANNOTATION_CATEGORIES),
+                    }
+                ),
+                "token_index": datasets.Sequence(datasets.Sequence(datasets.Sequence(datasets.Value("int32")))),
+                "total_words": datasets.Value("int32"),
+            }
+        )
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            supervised_keys=None,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+        """Returns SplitGenerators."""
+        data_dir = dl_manager.download_and_extract(_URLs)
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "filepath": os.path.join(data_dir["train+dev"], "task3", "train"),
+                    "split": "train",
+                },
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.TEST,
+                gen_kwargs={
+                    "filepath": os.path.join(data_dir["test"], "test-blind"),
+                    "split": "test",
+                },
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.VALIDATION,
+                gen_kwargs={
+                    "filepath": os.path.join(data_dir["train+dev"], "task3", "dev"),
+                    "split": "dev",
+                },
+            ),
+        ]
+
+    def _generate_examples(self, filepath, split):
+        """ Yields examples. """
+
+        def open_and_read(fp):
+            with open(fp, encoding="utf-8") as f:
+                return f.read().splitlines()
+
+        for id_, folder in enumerate(glob.glob(f"{filepath}/*")):
```

Comment:
```suggestion
        for id_, folder in enumerate(sorted(glob.glob(os.path.join(filepath, "*")))):
```
- make the order deterministic
- use os.path.join for windows users